src/shogun/clustering/GMM.cpp

```diff
@@ -185,6 +185,10 @@ float64_t CGMM::train_em(float64_t min_cov, int32_t max_iter, float64_t min_chan
 		pb.print_progress();
 		max_likelihood(alpha, min_cov);
 
+		this->observe<float64_t>(iter, "log_likelihood", "Log Likelihood", log_likelihood_cur);
+		this->observe<SGVector<float64_t>>(iter, "coefficients", "Mixture Coefficients", alpha);
```

Comment:
question (related to my "object" question earlier): is this vector copied? Because it will be modified in the next iteration

Reply:
Yes, it is copied.